tests/cephfs/cephfs_mirroring/validate_file_dir_stats_modification_on_mirrored_cluster.py

```diff
@@ -0,0 +1,439 @@
+        )
+        fs_mirroring_utils.add_path_for_mirroring(
+            source_clients[0], source_fs, subvol2_path
+        )
+        log.info("Add files into the path and create snapshot on each path")
+        snap1 = "snap_k1"
+        snap2 = "snap_f1"
+        log.info(f"{subvol1_path}")
+        log.info(f"{subvol2_path}")
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"touch {kernel_mounting_dir_1}{subvol1_path}/hello_kernel"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"touch {fuse_mounting_dir_1}{subvol2_path}/hello_fuse"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"mkdir {kernel_mounting_dir_1}{subvol1_path}/.snap/{snap1}"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"mkdir {fuse_mounting_dir_1}{subvol2_path}/.snap/{snap2}"
+        )
+        log.info("Validate the Snapshot Synchronisation on Target Cluster")
+        (
+            fsid,
+            asok_file,
+            filesystem_id,
+            peer_uuid,
+        ) = fs_mirroring_utils.validate_synchronization(
+            cephfs_mirror_node[0], source_clients[0], source_fs, 2
+        )
+        result_snap_k1 = fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node[0],
+            source_fs,
+            snap1,
+            fsid,
+            asok_file,
+            filesystem_id,
+            peer_uuid,
+        )
+        result_snap_f1 = fs_mirroring_utils.validate_snapshot_sync_status(
+            cephfs_mirror_node[0],
+            source_fs,
+            snap2,
+            fsid,
+            asok_file,
+            filesystem_id,
+            peer_uuid,
+        )
+        if result_snap_k1 and result_snap_f1:
+            log.info(f"Snapshot '{result_snap_k1['snapshot_name']}' has been synced:")
+            log.info(
+                f"Sync Duration: {result_snap_k1['sync_duration']} of '{result_snap_k1['snapshot_name']}'"
+            )
+            log.info(
+                f"Sync Time Stamp: {result_snap_k1['sync_time_stamp']} of '{result_snap_k1['snapshot_name']}'"
+            )
+            log.info(
+                f"Snaps Synced: {result_snap_k1['snaps_synced']} of '{result_snap_k1['snapshot_name']}'"
+            )
+            log.info(f"Snapshot '{result_snap_f1['snapshot_name']}' has been synced:")
+            log.info(
+                f"Sync Duration: {result_snap_f1['sync_duration']} of '{result_snap_f1['snapshot_name']}'"
+            )
+            log.info(
+                f"Sync Time Stamp: {result_snap_f1['sync_time_stamp']} of '{result_snap_f1['snapshot_name']}'"
+            )
+            log.info(
+                f"Snaps Synced: {result_snap_f1['snaps_synced']} of '{result_snap_f1['snapshot_name']}'"
+            )
+            log.info("All snapshots synced.")
+        else:
+            log.error("One or both snapshots not found or not synced.")
+        # scenario 1
```

Comment:
I would encourage to add loggers for separating the scenarios and logs for understanding and debugging purpose in future instead of comments in the test script.